os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -1980,6 +1980,7 @@ int wifi_scan_networks_with_ssid(int (results_handler)(char*buf, int buflen, cha
 	scan_buf_arg scan_buf;
 	int ret;
 
+	scan_buf.buf_len = 0;

```

Comment:
'SCAN_BUF.BUF_LEN' is assigned in the 1984 line, the next line.
Is there any reason to set it as 0 from here?